catalog/data_metastores_test.go

```diff
@@ -0,0 +1,92 @@
+package catalog
+
+import (
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go/service/catalog"
+	"github.com/databricks/terraform-provider-databricks/qa"
+)
+
+func TestMetastoresData(t *testing.T) {
+	qa.ResourceFixture{
+		Fixtures: []qa.HTTPFixture{
+			{
+				Method:   "GET",
+				Resource: "/api/2.0/accounts/testaccount/metastores",
```

Comment:
We use 2.0 https://github.com/databricks/databricks-sdk-go/blob/main/service/catalog/impl.go#L78